Course-feedback.md (#38)
introcompsys/spring2023: syllabus/grading.md

The tasks for these badges will be defined at the bottom of the notes for each class session *and* aggregated to badge-type specific pages on the left hand side. 

You can earn review and practice badges by:
1. Create an issue for the badge you plan to work on/Focus on a practice or prepare badge you plan to work on

> 2. Completing the tasks

Reviewer: should we remove this since it's redundant to 4?

3. Create a new branch for the badge through the main page of your KWL repository

> 4. Add a new file inside the new branch, and within that file have its contents be a task within that badge

Reviewer: ```suggestion
4. Add the new file from the badge instructions to the new branch, and within that file have its contents be a task within that badge (for example, answering questions, making notes, or output from running something) 
```

Reviewer: @victoriaryan2 does this revision still make sense?

5. After creating the file, a new button should appear for the option of creating a new pull request, which you should click and open

> 6. Then create the pull request

Reviewer: ```suggestion
6. Then create the pull request. Make the title match the badge name
```

7. Assign/merge any more files you may need into this pull request

> 8. You should then link the issue, by clicking the settings icon on the "development" tap(right side)
> 9. You should then request a review, by clicking the settings icon on the "reviewers" tap(right side)

Reviewer: ```suggestion
8. Link badge the issue, by clicking the settings icon on the "development" tab (right side)
9. To "submit," request a review, by clicking the settings icon on the "reviewers" tab (right side)
```
